tests/telemetry/test_telemetry.py

```diff
@@ -230,7 +230,7 @@ def test_sysuptime(duthosts, enum_rand_one_per_hwsku_hostname, ptfhost, gnxi_pat
     for line_info in system_uptime_info:
         if "total" in line_info:
             try:
-                system_uptime_1st = float(line_info.split(":")[1].strip())
+                system_uptime_1st = float(line_info.split(":")[1].strip().rstrip(','))
```

Comment:
>rstrip [](http://example.com/codeflow?start=74&length=6)

If the output is in json format, let's parse json. Not to parse ad-hoc.

Reply:
This is the output from client, not a clean json format output.

Performing GetRequest, encoding=JSON_IETF to X  with the following gNMI Path\n ------------------------- \n [elem {\n  name: \"proc\"\n}\nelem {\n  name: \"uptime\"\n}\n]\nThe GetResponse is below\n-------------------------\n\n{\n  \"total\": 5363.61,\n  \"idle\": 14432.27\n}\n-------------------------